src/buildstream/sandbox/_sandboxbuildboxrun.py

```diff
@@ -133,6 +133,15 @@ def _execute_action(self, action, flags):
             else:
                 stdin = subprocess.DEVNULL
 
+            if "casd-socket" in self.__config:
```

Comment:
This is probably a mistake: `self.__config` doesn't seem to exit.

Reply:
Hmm, is it the name mangling? self.__config belongs to grandparent of this class, Sandbox.

Reply:
Yeah, anything that starts with two underscores is private to a class (and python enforces it)